sdk/samples/iot/README.md

```diff
@@ -587,101 +572,40 @@ This section provides an overview of the different samples available to run and
+  X509 authentication is used to connect directly to Azure IoT Hub.
 
-    ```json
-    "2020-08-18T17:09:29-0700"
-    ```
+  The easiest way to interact with this sample from the service side is to use Azure IoT Explorer.  To use the sample:
 
-    The command will send back to the service a response containing the following JSON payload with updated values in each field:
+  - Follow the initial setup instructions described above.  Set the device's authentication using the steps from [IoT Hub X.509 Certificate Samples](#iot-hub-x509-certificate-samples).
+  - Install [Azure IoT Explorer](https://github.com/Azure/azure-iot-explorer/#plug-and-play).
+  - Download [the thermostat model](https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/samples/Thermostat.json) to a local directory.
+  - Build and run `paho_iot_pnp_sample`.
+  - Start Azure IoT Explorer and then:
+    - [Configure your hub](https://github.com/Azure/azure-iot-explorer/#configure-an-iot-hub-connection).  Once you've created your thermostat device, you should see it listed in the UX.
+    - Go to `IoT Plug and Play Settings` on the home screen, select `Local Folder` for the location of the model definitions, and point to the folder you downloaded the thermostat model.
+    - Go to the devices list and select your thermostat device.  Now select `IoT Plug and Play components` and then `Default Component`.
+    - You will now be able to interact with the Plug and Play device.
 
-    ```json
-    {
-      "maxTemp": 74.3,
-      "minTemp": 65.2,
-      "avgTemp": 68.79,
-      "startTime": "2020-08-18T17:09:29-0700",
-      "endTime": "2020-08-18T17:24:32-0700"
-    }
-    ```
-
-    No other commands are supported. If any other commands are attempted to be invoked, the log will report the method is not found.
-
-    </p>
-    </details>
-
-    <details><summary><b>Telemetry:</b></summary>
-    <p>
+Additional instructions for Azure IoT Explorer, including screenshots, are available [here](https://github.com/Azure/azure-iot-explorer/#plug-and-play).
+  
+### IoT Plug and Play with Provisioning Sample
 
-    Device sends a JSON message with the property name `temperature` and a `double` value for the current temperature.
+- *Executable:* `paho_iot_pnp_with_provisioning_sample`
 
-    </p>
-    </details>
+  This sample has the same functionality as the `paho_iot_pnp_sample` but uses the Azure Device Provisioning Service for authentication. The same steps above may be followed for interacting with the sample, but you must instead set the environment variables described in [IoT Provisioning X.509 Certificate Sample](#iot-provisioning-x509-certificate-sample) for authentication.
 
-  </details>
 
-### IoT Hub Plug and Play Multiple Component Sample
+### IoT Plug and Play Multiple Component Sample
```

Comment:
I'm confused.  It looks like you are taking out the multiple component sample in some places, but it exists others.  I'm not sure which one you meant to do.

Reply:
If keeping Multiple Component Sample, it should go before the "with Provisioning Sample" in all PnP ordered spots  -- readme, cmake, etc.

Reply:
Yup, the multiple components is a "comming attraction".  I know there's much to cleanup around that too which is why it'll be taken care of in PR4B of this.

Reply:
Oh, seeing this now.  That is why you have left this sample in the cmake.  please ignore my other comments on that. thanks.